### DISCUSSION TITLE
How can I fix it?

### DISCUSSION THREAD

**Discussion by CoolDude4343** [mupen64plus-ae/mupen64plus-ae#1157] [Q&A]
In the game Resident Evil 2 there is a problem where the character is always on the top layer, causing the scenery to always be behind, how can I solve this? If possible. Thanks

**Comment by fzurita** [comment]
You will need a newer phone.

Select the game, go to settings, and change the emulation profile to GLideN64-Very-Accurate. Software render, will be even more accurate, but it will be more blurry.

**Reply by CoolDude4343** [reply]
Thank you!
